src/Makevars

```diff
@@ -1,4 +1,5 @@
 PKG_CXXFLAGS += -DRCPP_USE_UNWIND_PROTECT
+KG_CXXFLAGS += $(shell ${R_HOME}/bin/Rscript -e "RcppParallel::CxxFlags()")
```

Comment:
There's a typo, an initial P is missing. Also, where did you find this information? [The documentation](https://rcppcore.github.io/RcppParallel/#r_packages) doesn't mention this.

Reply:
> There's a typo, an initial P is missing. 

Oh good catch, sorry about that!

> Also, where did you find this information? [The documentation](https://rcppcore.github.io/RcppParallel/#r_packages) doesn't mention this.

The `CxxFlags` function didn't previously have much use for packages, but the TBB headers needs additional compilation flags on Windows ARM64